roles/download/defaults/main.yml

```diff
@@ -10,6 +10,9 @@ skip_downloads: false
 download_run_once: False
 download_compress: 1
 
+# if this is set to true will download container
+download_container: True
```

Comment:
why this new default ?

Reply:
Originally, download_container is not defined and write code as download_container|default(true) each time. It's not clean for me.  If you prefer download_container|default(true) style, I'll rollback this modification.

Reply:
no, that's better